Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -2028,6 +2028,7 @@
 				"en-GB",
 				co,
 				tt,
+				et,
```

Comment:
There is only this change, while after adding a locale in firefox we see a lot more changes in the Client.xcodeproj/project.pbxproj file
https://github.com/mozilla-mobile/firefox-ios/pull/11417/files#diff-053ad7baef0cd20365abec6f2d6133fa7eed9440d908c9f92e1319c8d610d17f)./
https://github.com/mozilla-mobile/firefox-ios/pull/11417/files#diff-053ad7baef0cd20365abec6f2d6133fa7eed9440d908c9f92e1319c8d610d17fR3894